Dockerfile

```diff
@@ -6,12 +6,17 @@ COPY composer.lock /usr/src/app/
 
 RUN apk --update add git php-common php-xml php-dom php-ctype php-iconv php-json php-pcntl php-phar php-openssl php-opcache php-sockets curl && \
     curl -sS https://getcomposer.org/installer | php && \
-    /usr/src/app/composer.phar install && \
-    apk del build-base && rm -fr /usr/share/ri
+    /usr/src/app/composer.phar install
+RUN apk --update add build-base ruby-dev ruby ruby-bundler && gem install httparty --no-rdoc --no-ri
```

Comment:
I split these up into several RUN methods so when we change Ruby it doesn't also re-fetch PHP. Let me know if we prefer it as one continuous line.